conversion_helpers_test.go

```diff
@@ -395,6 +395,7 @@ var _ = Describe("Resources", func() {
 						},
 					},
 					LogRateLimitBytesPerSecond: -1,
+					VolumeMountedFiles:         []executor.VolumeMountedFiles{{Path: "/redis/username", Content: "redis_user"}},
```

Comment:
❓ Can you add a test for when volumeMountedFiles are no provided, that the NewRunRequestFromDesiredLRP does not error?